packages/flutter/lib/src/widgets/primary_scroll_controller.dart

```diff
@@ -16,6 +16,19 @@ import 'scroll_controller.dart';
 /// This mechanism can be used to provide default behavior for scroll views in a
 /// subtree. For example, the [Scaffold] uses this mechanism to implement the
 /// scroll-to-top gesture on iOS.
+///
+/// Another default behavior handled by the PrimaryScrollController is default
+/// [ScrollAction]s. If a ScrollAction is not handled by an otherwise focused
+/// part of the application, the ScrollAction will be evaluated using the scroll
+/// view associated with this ScrollController, for example, when executing
```

Comment:
"this ScrollController" - it's not guaranteed that "this" scrollcontroller is used. The action uses whatever PrimaryScrollController it can find in its context, no?

Reply:
Tweaked.